lib/vm/src/context.rs

```diff
@@ -11,12 +12,18 @@ use cosmwasm::traits::{Api, Storage};
 use crate::memory::{read_region, write_region};
 use cosmwasm::types::{CanonicalAddr, HumanAddr};
 
+/// An undocumented, unstable constant. This can change at any time. Be warned.
+static WRITE_REGION_ERROR: i32 = -1001;
```

Comment:
Good idea.
Maybe we should make constants for the "-1" errors as well.

Reply:
Yeah, we could have other values for `from_utf8`, `api.canonical_address` and `api.human_address` fails.

-1 is like a default error that means different things in different methods. In order to avoid collisions with random negative values, I started with 1001.